### PR TITLE
added newline handling for Windows

### PR DESCRIPTION
**Pull request by jworcestBAH**
Added newline specification to prevent extra lines on Windows, and cut some duplicate code in csv writer

### CODE REVIEW

**Comment by jworcestBAH** [comment]
Jenkins failing to reach job manager, will talk with Katie tomorrow about that.

**Comment by klrBAH** [comment]
[run tests]

File: dataactvalidator/filestreaming/csvLocalWriter.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(self,filename,header) :
         header - list of strings for the header
 
         """
-        self.stream = open(filename,"w")
+        self.stream = open(filename,"w", newline = '')
```
Comment (bsweger) [inline dataactvalidator/filestreaming/csvLocalWriter.py]: Had to read up on this a bit to see what `newline = ''` does. Great catch!

**Comment by bsweger** [comment]
@jworcestBAH The code changes look good. If you ping me when Jenkins/job manager is sorted, I can merge.

**Comment by klrBAH** [comment]
[run tests]

**Comment by klrBAH** [comment]
@jworcestBAH @bsweger, unit tests have been re-run and completed successfully.

**Pull request merged**